### DISCUSSION TITLE
why is uNetworking/uWebSockets.js not included in the benchmarks shared by fastify?

### DISCUSSION THREAD

**Discussion by Dhruv-Garg79** [fastify/fastify#5017] [General] [answered]
uNetworking/uWebSockets.js is another web server for node.js which claims to be much faster than fastify. It would have been better if it was included in the benchmarks published by fastify

**Comment by jsumners** [comment]
https://github.com/search?q=org%3Afastify+uwebsockets&type=pullrequests

**Reply by Dhruv-Garg79** [reply]
do you know why it is not getting merged?
If the issue is just use of old package, I can update it and make a push to same PR

**Reply by mcollina** [reply, marked as answer]
because ultimately uwebsocket is not Node.js. It does not use the Node.js classes for HTTP, but it's a different implementation in C++. For the same reason why we do not add Bun or Deno there.
